src/scss/cs/homepage.scss

```diff
@@ -284,7 +289,6 @@
       text-align: center;
 
       @media (max-width: 768px) {
-       
```

Comment:
We won't need the Media Query from lines 291-292 because it's not in use anymore :)